lib/controllers/ServiceFabrikAdminController.js

```diff
@@ -24,24 +24,42 @@ class ServiceFabrikAdminController extends BaseController {
   }
 
   updateDeployment(req, res) {
+    const self = this;
     const redirect_uri = _.get(req.query, 'redirect_uri', '/admin/deployments/outdated');
-    return this.fabrik
-      .createOperation('update', {
-        deployment: req.params.name,
-        username: req.user.name,
-        arguments: req.body
-      })
-      .invoke()
-      .then(body => {
-        res.format({
-          html: () => res
-            .redirect(303, redirect_uri),
-          default: () => res
-            .status(200)
-            .send(body)
-        });
+    const allowForbiddenManifestChanges = req.body.forbidden_changes === undefined ? true : req.body.forbidden_changes;
+    const deploymentName = req.params.name;
 
-      });
+    function updateDeployment() {
+      return self.fabrik
+        .createOperation('update', {
+          deployment: deploymentName,
+          username: req.user.name,
+          arguments: req.body
+        }).invoke()
+        .then(body => {
+          res.format({
+            html: () => res
+              .redirect(303, redirect_uri),
+            default: () => res
+              .status(200)
+              .send(body)
+          });
+        });
+    }
+    logger.info(`Forbidden Manifest flag set to ${allowForbiddenManifestChanges}`);
+    if (allowForbiddenManifestChanges === 'false') {
```

Comment:
shouldn't this be boolean comparison ? does this work ?

Reply:
yes this works, i specifically tested this.
Data passed in curl is interpreted as text.
But i guess i can remove this confusion if i JSON.parse req.body [here]
 (https://github.com/SAP/service-fabrik-broker/pull/57/files/2539bc794c0413b831d5f5ed0b237e7cc6bc7a9f#diff-7f13ae4ec4eb9672a001b5943664cbcdR29)